oci/archive/oci_transport.go

```diff
@@ -194,7 +194,7 @@ func (t *tempDirOCIRef) deleteTempDir() error {
 
 // createOCIRef creates the oci reference of the image
 func createOCIRef(image string) (tempDirOCIRef, error) {
-	dir, err := ioutil.TempDir("/var/tmp", "oci")
+	dir, err := ioutil.TempDir("", "oci")
```

Comment:
Why does this PR need this change? It seems unrelated.

(Using `/var/tmp` on is necessary on Linux because `/tmp` may be `tmpfs`, inappropriate for storing multi-gigabyte container images.)

Reply:
This is indeed unrelated, however, I came across this while rebasing my original pull request. OCI is one of the few transports which would work across OS. /var/tmp will of course not work on Windows. IMO the basedir should not be specified, but I can revert this, if that means we can get this merged.

Reply:
Yes, please split this out. Having as much as possible work on Windows would be very nice, but the fix should

- Be in a clearly separate commit, with an explanation of what is fixing. (At least I, with my Linux blinders on, could not guess that this improves Windows.)
- Not break Linux/macOS to fix Windows.
- Ideally, be comprehensive. See the other two instances of `temporaryDirectoryForBigFiles`; I guess this would end up as a tiny sub-package.

Reply:
@mtrmac - I created a new issue - https://github.com/containers/image/issues/368. I agree this should be properly tracked since it is not directly related to this pull request. 

Not sure whether I have the bandwidth to work on this, but either way I would like to see my others pull requests merged/addressed first. We need them merged in order to start using containers/image in Minishift.

Reply:
> @mtrmac - I created a new issue - #368.

Thanks!